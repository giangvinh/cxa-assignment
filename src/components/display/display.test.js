import React from 'react';
import { shallow } from 'enzyme';
import Display from './display.component';

it('should render Display component', () => {
  expect(shallow(<Display />)).toMatchSnapshot();
});
