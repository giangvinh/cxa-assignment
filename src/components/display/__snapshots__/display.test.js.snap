// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Button Panel component 1`] = `ShallowWrapper {}`;

exports[`should render Display component 1`] = `ShallowWrapper {}`;
